src/app/not-found-page/not-found-page.component.css

```diff
@@ -0,0 +1,117 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
```

Comment:
**suggestion:** fonts should be kept inside the project as assets (we have `assets` folder) otherwise if you want to have them taken from a CDN, you should reference them in the `index.html`